app/src/main/kotlin/com/wire/android/ui/home/conversations/call/ConversationListCallViewModel.kt

```diff
@@ -101,8 +101,10 @@ class ConversationCallViewModel @Inject constructor(
             // valid conversation is a conversation where the user is a member and it's not deleted
             val validConversation = when (conversationDetailsResult) {
                 is ObserveConversationDetailsUseCase.Result.Success -> {
-                    !(conversationDetailsResult.conversationDetails is ConversationDetails.Group &&
-                            !(conversationDetailsResult.conversationDetails as ConversationDetails.Group).isSelfUserMember)
+                    !(
+                            conversationDetailsResult.conversationDetails is ConversationDetails.Group
+                                    && !(conversationDetailsResult.conversationDetails as ConversationDetails.Group).isSelfUserMember
+                            )
```

Comment:
This is a bit hard to read.
Could you please extract to more readable vals before putting into the if?

Like 
```kotlin
val isGroupConversation = ...
```